test/benchmark.cc

```diff
@@ -53,6 +53,8 @@ CDADA_MAP_CUSTOM_TYPE_DECL(test_u3552_t);
 CDADA_SET_CUSTOM_TYPE_DECL(test_u3552_t);
 CDADA_STACK_CUSTOM_TYPE_DECL(test_u3552_t);
 
+#ifdef __amd64__
```

Comment:
I think this is also valid for x86

Reply:
There are many preprocessor macros defined for both `amd` and `x86`. See https://sourceforge.net/p/predef/wiki/Architectures/. It seemed a bit of overkill to add them all. What do you think?

Reply:
Certainly the predefined variabless preprocessor path seems to convoluted.

It might be cleaner to run an `AC_COMPILE_IFELSE` in `configure.ac` with a very simple program trying to use `__rdtsc`, and then create a MACRO as a result. If that would be in a separate m4, e.g. `config_m4/rdtsc.m4`, it would be ideal

Reply:
Done. Please see code diffs.